tests/dialects/test_comb.py

```diff
@@ -14,3 +17,38 @@ def test_icmp_incorrect_comparison():
         # 'slet' is an invalid comparison operation
         _icmp_op = ICmpOp(a, b, "slet")
     assert e.value.args[0] == "Unknown comparison mnemonic: slet"
+
+
+def test_comb_concat_builder():
+    a = TestOp(result_types=[IntegerType(5)])
+    b = TestOp(result_types=[IntegerType(3)])
+    c = TestOp(result_types=[IntegerType(1)])
+    foo = TestOp(result_types=[TestType("foo")])
```

Comment:
Can you replace these with `TestSSAValue(IntegerType(_))` like in the previous example?
It's just to stay consistent with the rest of the codebase.

Reply:
I cannot believe I missed this. The infrastructure in this project is just so fancy.